packages/omg-js-rootchain/src/txUtils.js

```diff
@@ -48,24 +46,15 @@ async function sendTx ({ web3, txDetails, privateKey, callbacks }) {
         .on('error', reject)
     })
   }
-  return web3.eth.sendSignedTransaction(signedTx.rawTransaction).catch(err => {
-    let transactionHash
-    try {
-      transactionHash = JSON.parse(err.message.replace('Transaction has been reverted by the EVM:', '')).transactionHash
-    } catch (parseError) {
-      throw (err)
-    }
-    ethErrorReason({ web3, hash: transactionHash }).then(() => {
-      throw (err)
-    })
-  })
+  return web3.eth.sendSignedTransaction(signedTx.rawTransaction)
```

Comment:
since we remove the parsing here, agree it is weird throwing in a catch. but after removing this, where does the error passing go ?

Reply:
its returned to the caller to handle the error.
```
try {
  await rootChain.deposit()
} catch (err) {
// handle failure how u want
  ethErrorReason(hash)
}
```